packages/cli/src/commands/dev.ts

```diff
@@ -50,6 +62,16 @@ async function storeDev(rootDir: string, tmpDir: string) {
   })
 }
 
+function copyGenerated(from: string, to: string) {
+  try {
+    cpSync(from, to, { recursive: true, force: true })
+
+    return { success: true }
+  } catch (err) {
+    return { success: false }
```

Comment:
question: an error to copy would abort the command run?

Reply:
It should not. But I believe you asked this question because the code is returning this and we're not using it anywhere else. I'll adjust the code to warn the user of the impact of the failure to copy.

Reply:
Thanks!